### ISSUE TITLE
on Amazon Linux 2 ssh-find-agent's output includes "garbage"

### ISSUE THREAD

**Issue by wwalker** [wwalker/ssh-find-agent#35]
I suspect it is a bash version issue.
```
$ eval "$(ssh-find-agent)"
-bash: syntax error near unexpected token `RSA'
$ printf "<%s>\n" "$(ssh-find-agent)"
<export SSH_AUTH_SOCK=/tmp/ssh-uJ7Ualz2Tu/agent.28347   #1)     wwalker@polonium (RSA)
waywalke@polonium (RSA)   >
$ bash --version
GNU bash, version 4.2.46(2)-release (x86_64-koji-linux-gnu)
Copyright (C) 2011 Free Software Foundation, Inc.
License GPLv3+: GNU GPL version 3 or later <http://gnu.org/licenses/gpl.html>

This is free software; you are free to change and redistribute it.
There is NO WARRANTY, to the extent permitted by law.
$
```

**Comment by wwalker** [comment]
This was a design flaw that is triggered in certain conditions.

It is fixed in the v1.0 branch (which seems functional now, but isn't officially released

**Comment by wwalker** [comment]
Closing.
